roles/cdi/tasks/version.yml

```diff
@@ -0,0 +1,15 @@
+---
+# 'version_compare' changes to 'version' in ansible-2.5
+- name: "Setting cdi_release_tag to 0.5 if cdi_release_tag: {{ cdi_release_tag }} is >= 1.0 and kubevirt_version: {{ kubevirt_version }} < 0.7.0-alpha.4"
+  set_fact: cdi_release_tag="0.5"
+  when: "{{ '1.0' is version_compare(cdi_release_tag, '<=') }}
+  	and {{ '0.7.0-alpha.4' is version_compare(kubevirt_version, '>') }}"
+
+# 'version_compare' changes to 'version' in ansible-2.5
+- name: "Setting cdi_release_tag to 1.0 if cdi_release_tag: {{ cdi_release_tag }} is < 1.0 and kubevirt_version: {{ kubevirt_version }} >= 0.7.0-alpha.4"
+  set_fact: cdi_release_tag="1.0"
+  when: "{{ '1.0' is version_compare(cdi_release_tag, '>') }}
+  	and {{ '0.7.0-alpha.4' is version_compare(kubevirt_version, '<=') }}"
+
+- debug:
```

Comment:
Do we have to add more of these each time we add a new incompatible release?  This is going to get ugly really fast.  Why not just set the defaults to a working combination for the latest release and hard code the APBs to use the working combination corresponding to the synched release tag?

Reply:
Default will be for working combinations - this functions as 1) a backstop for incorrect versions 2) a way for the kubevirt-apb to consume cdi 1.0 and 0.7.0-alpha.4 before kubevirt-ansible official moves to that release (PRs are currently blocked on CI https://github.com/kubevirt/kubevirt-ansible/pull/306 and https://github.com/kubevirt/kubevirt-ansible/pull/295).  The hardcoded value that the APBs consume is the default value, so it's blocked on https://github.com/kubevirt/kubevirt-ansible/pull/306 and https://github.com/kubevirt/kubevirt-ansible/pull/295 too.